Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -645,13 +645,51 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
     }
 
     if node.label.isMissing {
+      let replaceTokens: [TokenSyntax] = [node.unexpectedBetweenLabelAndColon].compactMap { $0?.tokens(viewMode: .sourceAccurate) }.flatMap { $0 }
```

Comment:
Can’t this just be 

```suggestion
      let replaceTokens = node.unexpectedBetweenLabelAndColon.compactMap { $0?.tokens(viewMode: .sourceAccurate) }
```

Reply:
But we still need to cast `[TokenSequence]?` to `[TokenSyntax]`. I think mine would be simpler.